pkg/daemon/daemon.go

```diff
@@ -200,6 +210,41 @@ func (dn *Daemon) applyNodePtpProfile(runID int, nodeProfile *ptpv1.PtpProfile)
 	return nil
 }
 
+func populatePtp4lConf(tomlContent *string, output *ptp4lconf) {
+	lines := strings.Split(*tomlContent, "\n")
+	var current string;
+	output.sections = map[string]ptp4lconfSection{}
+
+	for _, line := range lines{
+		if strings.HasPrefix(line, "["){
+			current = line
+			output.sections[current] = *new(ptp4lconfSection)
```

Comment:
I don't understand this comment.  I am allocating the new section, that will contain either the global or an interface config section.

Reply:
it's more of a cosmetic change, we usually use &type{} instead of new() for new objects.
so in this case you should use and it won't change anything
```
output.sections[current] = &ptp4lconfSection{}
```

Reply:
updated